app/sidekiq/lighthouse/submit_career_counseling_job.rb

```diff
@@ -49,5 +57,20 @@ def send_confirmation_email(user_uuid)
         }
       )
     end
+
+    def self.trigger_failure_events(claim)
```

Comment:
Unless you plan to use this outside of this could just be a private method since you're only using it in the confines of the class

Reply:
Since it's a class method, private won't do much for it and I get a rubocop exception. The reason it isn't an instance is because it can't be accessed inside of the retries exhausted block unless it's a class method.

Reply:
Sounds good